.github/workflows/package.yml

```diff
@@ -58,7 +58,7 @@ jobs:
           sudo apt-get install -y libudev-dev
 
       - name: Install Lerna
-        run: yarn global add lerna
+        run: yarn global add lerna@5.6.2
```

Comment:
> An explanatory comment is needed to clarify the reason for the existence of this version number. Otherwise, this code may be removed by someone else in the future.

At the moment, we are using lerna version 5.6.2. In the GitHub action, no specific version of lerna is specified, so it may try to install the latest version, which could cause errors. For now, I have temporarily established an agreement on the version of lerna we will be using. Once we upgrade, we will remove this version constraint. I will leave a comment here.